<ui version="4.0" >
 <class>KbfxConfigDlgThemes</class>
 <widget class="QWidget" name="KbfxConfigDlgThemes" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="sizeConstraint" >
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="lbl_menu_top" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #90b8d7, stop: 1 #0c669d);
color: #ffffff;</string>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">KBFX Themes Folder&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Themes Folder Location</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Please select the source folder where your themes reside. (default /usr/share/apps/kbfx/skins)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLineEdit" name="lineEdit" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text" >
             <string>Locate</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >
              <normaloff>:/buttons/images/folder.png</normaloff>:/buttons/images/folder.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="sizeConstraint" >
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="lbl_menu_top_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #90b8d7, stop: 1 #0c669d);
color: #ffffff;</string>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">KBFX Themes Installed&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Installed Themes Preview &amp;&amp; Selection</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QListWidget" name="listWidget" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text" >
            <string>2panels</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>default</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title" >
           <string>Selected Them Preview</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9" >
           <item>
            <widget class="QLabel" name="label_2" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;img src=":/ZTEST/images/low_menu1.jpg" />&lt;/p>&lt;/body>&lt;/html></string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" >
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8" >
               <item>
                <widget class="QPushButton" name="pushButton_5" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>85</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>&amp;Theme Info</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_6" >
                 <property name="minimumSize" >
                  <size>
                   <width>85</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>&amp;Get Themes</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6" >
               <item>
                <widget class="QPushButton" name="pushButton_3" >
                 <property name="minimumSize" >
                  <size>
                   <width>85</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>&amp;Install Theme</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_4" >
                 <property name="minimumSize" >
                  <size>
                   <width>85</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>D&amp;elete Selected</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" >
          <item>
           <widget class="QGroupBox" name="groupBox_4" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title" >
             <string>Personal Image</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10" >
             <item>
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">This is your personal image&lt;br />displayed at the KBFX menu.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;img src=":/ZTEST/images/low_menu2.jpg" />&lt;/p>&lt;/body>&lt;/html></string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2" >
               <property name="text" >
                <string>&amp;Select New Image</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Max size 48 x 48</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_7" >
               <property name="text" >
                <string>The&amp;me's Default</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title" >
             <string>Misc. Options</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7" >
             <item>
              <widget class="QCheckBox" name="checkBox" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Show old themes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Watch System Installs</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
